zktest/src/archive/test2/B85-ZK-3926.zul

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+B85-ZK-3926.zul
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Wed Jun 13 14:15:53 CST 2018, Created by charlesqiu
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+<zk>
+	<label multiline="true" pre="true"><![CDATA[
+		1. Click open arrow on north
+		2. You should see the north open with animation
```

Comment:
Because you wrote a log warning, You can write
3. check "The library-property setting: "org.zkoss.zul.borderlayout.animation.disabed" was changed to "org.zkoss.zul.borderlayout.animation.disabled", please use the new one instead." on console.